sailfishusermanagerinterface.h

```diff
@@ -12,10 +12,20 @@
 
 #include <QString>
 #include <QDBusArgument>
+#include <QDBusError>
 
 #define SAILFISH_USERMANAGER_DBUS_INTERFACE "org.sailfishos.usermanager"
 #define SAILFISH_USERMANAGER_DBUS_OBJECT_PATH "/"
 
+#define SailfishUserManagerErrorBusy "org.sailfishos.usermanager.Error.Busy"
```

Comment:
These are a bit inconsistent since some of these have word Error and some don't.

Reply:
Yep